packages/explicit-constructed-response/src/main.js

```diff
@@ -54,29 +61,51 @@ export class Main extends React.Component {
 
   render() {
     const { showCorrectAnswer, value } = this.state;
-    const { classes, mode, prompt, rationale, teacherInstructions, note, showNote, env, maxLengthPerChoice } = this.props;
+    const {
+      classes,
+      mode,
+      prompt,
+      rationale,
+      teacherInstructions,
+      note,
+      showNote,
+      env,
+      animationsDisabled,
+      alwaysShowCorrect,
+      maxLengthPerChoice
+    } = this.props;
     const { role } = env || {};
     const displayNote = (showCorrectAnswer || mode === 'view' && role === 'instructor') && showNote && note;
+    const mainClasses = alwaysShowCorrect ? classNames(classes.mainContainer, classes.noBorderColor)
```

Comment:
You could also use this format: `classNames('foo', { bar: true }); // => 'foo bar'`